include/glow/IR/IR.h

```diff
@@ -320,6 +320,12 @@ class IRFunction final : public Named {
   /// \returns the variable map.
   VariableMap &getVariableMap() { return variableMap_; }
 
+  /// Returns a list of constants associated with function.
+  std::list<const Constant *> getConstants() const;
+
+  /// Returns a list of placeholders associated with the function.
+  std::list<const Placeholder *> getPlaceholders() const;
```

Comment:
Why not use `PlaceholderList`?

Reply:
It's a list of const Constants, ConstList is just a list of Constants. :D